### ISSUE TITLE
Req. : Add Vertical Line Between Columns in Word Document

### ISSUE THREAD

**Issue by wcgJm28** [PHPOffice/PHPWord#2679]
Please implement a feature that adds a vertical line between two columns in the Word document generated by the system, similar to the provided example image. This line should serve as a visual separator between the content of the columns. The line should:

1. Be correctly aligned between the columns.
2. Have customizable styling (e.g., thickness, color, length).
3. Remain consistent across all pages of the document.
4. Be adaptable to varying numbers of columns (colsNum), and properly adjust its position to maintain balance between multiple columns.
5. This feature is important for better readability and clear separation of content in documents with multiple columns.

![image](https://github.com/user-attachments/assets/35fb2eed-00a6-4f49-9efb-760c85336e95)

**Comment by Progi1984** [comment]
@wcgJm28 Have you got a sample file for analysis ?

**Comment by wcgJm28** [comment]
Yes, I have created a sample file for analysis. The file name is sample file.docx. Additionally, I have generated another file using the PhpWord package, which is named Created File From Package.docx.

[Created FIle From Package.docx](https://github.com/user-attachments/files/17022617/Created.FIle.From.Package.docx)
[Sample File.docx](https://github.com/user-attachments/files/17022618/Sample.File.docx)

**Comment by hamdan7m** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
